tests/python/relay/test_pass_partition_graph.py

```diff
@@ -201,8 +202,11 @@ def check_vm_result():
         exe = runtime.vm.Executable.load_exec(code, lib)
         vm = runtime.vm.VirtualMachine(exe)
         vm.init(ctx)
-        out = vm.run(**map_inputs)
-        tvm.testing.assert_allclose(out.asnumpy(), result, rtol=tol, atol=tol)
+        outs = vm.run(**map_inputs)
+        outs = outs if len(outs) > 1 else [outs]
```

Comment:
is `len(outs)` oaky? It doesn't necessarily be an ADTObj/array, right? what if it is an objectref (i.e. NDArray in this case)?

Reply:
Yeah the test failed because of this. Changing to `isinstance(outs, runtime.container.ADT)`